reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -18,18 +23,33 @@
   color: #133a53;
 }
 
+.darkmode .heatmap table th {
+  color: #f2f2f2;
+}
+
+.table th {
+  border-top: none;
+}
+
+.darkmode .heatmap table th {
+  background-color: rgb(64, 64, 64);
```

Comment:
This could be merged with the block up above line 26

Reply:
✅

Reply:
This doesn't appear to be fixed

Reply:
I think this is fixed now